src/utils/rari.js

```diff
@@ -526,6 +538,23 @@ export const getRariWithdrawTransactionData = async (
         continue;
       }
 
+      if (_0xdata[0].some(order => EthersBigNumber.from(order.takerAssetAmount).gt(inputTokenBN))) {
+        _0xdata = await get0xSwapOrders(
+          RARI_TOKENS[inputCandidate.currencyCode].address,
+          token.symbol === ETH
+            ? WETH
+            : token.address,
+          inputTokenBN,
+        ).catch(error => {
+          reportErrorLog("Rari service failed: Can't get 0x swap orders", { error });
+          return null;
+        });
+      }
+
+      if (!_0xdata) {
+        continue;
```

Comment:
Not sure if we should log this or this is something too common?

Reply:
it'll be logged in `get0xSwapOrders` function